easybuild/easyconfigs/g/GeneTorrent/GeneTorrent-3.8.7-foss-2015a.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+
+name = 'GeneTorrent'
+version = '3.8.7'
+
+homepage = 'https://cghub.ucsc.edu/index.html'
+description = """ GeneTorrent is a command-line tool required to download datasets
+held in repositories at the Cancer Genomics Hub. """
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['https://cghub.ucsc.edu/software/downloads/GeneTorrent/%(version)s/']
+sources = [SOURCE_TAR_GZ]
+
+configopts = '--with-boost=$EBROOTBOOST --with-xerces=$EBROOTXERCESMINCPLUSPLUS --with-xqilla=$EBROOTXQILLA'
+
+dependencies = [
+    ('Xerces-C++', '3.1.2'),
+    ('cURL', '7.44.0'),
+    ('XQilla', '2.3.0', '-Xerces-C++-3.1.2'),
+    ('Boost', '1.55.0', '-Python-2.7.9'),
```

Comment:
bumping this to 1.58.0 makes sense, since it's very common

we have easyconfigs for 1.59.0 (latest) too, but I've seen problems with that recently (they probably screwed up their API again i n1.59.0)